<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalkBrader</class>
 <widget class="QMainWindow" name="TalkBrader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>554</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="chatPage">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::InputMethodHint::ImhNone</set>
        </property>
        <property name="text">
         <string>No Chat Avilable. Click + Icon To Chat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>479</x>
          <y>40</y>
          <width>91</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="addFriendPage">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>90</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchBar">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>361</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="searchB">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>search</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>601</width>
          <height>341</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>582</width>
           <height>231</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="registerPage">
       <widget class="QLineEdit" name="usernameBarRigster">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>110</y>
          <width>201</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordBarRigster">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>201</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>410</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Already Have An Account?</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QPushButton" name="registerBRegister">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>260</y>
          <width>121</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loginBRegister">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>400</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordBarRigster_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>201</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Re-Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>441</width>
          <height>20</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="signoutPage">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>140</y>
          <width>511</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Log in to an existing account or create a new one.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="registerB">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>300</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loginB">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>220</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>180</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="usernameBar">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>180</y>
             <width>201</width>
             <height>28</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="passwordBar">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>220</y>
             <width>201</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>220</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
          <widget class="QPushButton" name="loginB_2">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>290</y>
             <width>121</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>400</y>
             <width>231</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dont Have An Account?</string>
           </property>
          </widget>
          <widget class="QPushButton" name="registerB_2">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>390</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>90</y>
             <width>441</width>
             <height>20</height>
            </rect>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
